### PR TITLE
Add documentation for findBy

### PR DESCRIPTION
**Pull request by davewasmer**
TIL mirage has `findBy`, and I can stop writing `users.where({ username }).models[0]` 😜

### CODE REVIEW

File: docs/v0.2.x/models.md [hunk 1]
```diff
@@ -91,6 +91,14 @@ let post = blogPosts.find(1);
 let posts = blogPosts.find([1, 3, 4]);
 ```
 
+### findBy(*query*)
+
+Returns the first model in the database that matches the key-value pairs in the `query` object. Note that a string comparison is used. `query` is a POJO.
+
+```js
+let posts = blogPosts.where({ published: true });
```
Comment (samselikoff) [inline docs/v0.2.x/models.md]: is it supposed to be `findBy` instead of `where`

Comment (davewasmer) [inline docs/v0.2.x/models.md]: Derp :man_facepalming:

**Comment by locks** [comment]
Psst, what's a POJO?

**Comment by samselikoff** [comment]
thanks! ❤️

**Pull request merged**